README.md

```diff
@@ -32,7 +32,7 @@ You can select your profile/region by &larr;, &uarr;, &rarr; &darr; and filter b
 ## Build
 
 ```bash
-go build
+make all

```

Comment:
It means you PR#3 is useless, right?

Reply:
@Issif No, I would like PR #3 to be merged also to support binary release distribution.